ProcessLib/ProcessOutput.cpp

```diff
@@ -215,8 +216,7 @@ void doProcessOutput(std::string const& file_name,
 
     // Write output file
     DBUG("Writing output to \'%s\'.", file_name.c_str());
-    MeshLib::IO::VtuInterface vtu_interface(&mesh, vtkXMLWriter::Binary,
-                                            compress_output);
+    MeshLib::IO::VtuInterface vtu_interface(&mesh, data_mode, compress_output);
```

Comment:
Do we have to specify the precision for Ascii output?

Reply:
It seems to me the ascii output uses `%g` from printf, so it is switching btw. `X.Y` and `X.YeZ` forms while outputting 11 significant digits for X and Y together.
I don't find any documentation about setting the output precision in vtk docu...
I guess the ascii output format is given by vtk and not customizable.

Reply:
Some example:
```xml
<DataArray type="Float64" Name="sigma_xy" format="ascii" RangeMin="-7.4359310551e-15" RangeMax="2.5418831431e-15">
-1.2844131096e-15 -2.8590975224e-15 ... 2.2671674321e-15 -6.4106780642e-15
</DataArray>
<DataArray type="Float64" Name="sigma_yy" format="ascii" RangeMin="-165.79792808" RangeMax="-165.79792808">
-165.79792808 -165.79792808 ...  -165.79792808 -165.79792808
</DataArray>
```

Reply:
Comparing ascii and binary outputs of same simulation:
```sh
➜  ogs/d bin/vtkdiff -a displacement -b displacement a.vtu cube_1e0_pcs_0_ts_100_t_0.010000.vtu 
Comparing data array `displacement' from file `a.vtu' to data array `displacement' from file `cube_1e0_pcs_0_ts_100_t_0.010000.vtu'.
Computed difference between data arrays:
abs l1 norm      = 3.093700426046175e-14
abs l2-norm^2    = 1.196238634558976e-28
abs l2-norm      = 1.093726946984016e-14
abs maximum norm = 3.866969678495424e-15

rel l1 norm      = 9.043003552305743e-11
rel l2-norm^2    = 1.022159699263067e-21
rel l2-norm      = 3.197123237010214e-11
rel maximum norm = 1.130371587200067e-11
```

Reply:
Thanks for checking this. Your observations seem to be correct:
https://gitlab.kitware.com/vtk/vtk/commit/47248b0d194a299e35c67af890fa350121e343c2
https://gitlab.kitware.com/vtk/vtk/blob/master/IO/XML/vtkXMLWriter.cxx#L828
and also all "stream"-related methods are protected:
http://www.vtk.org/doc/nightly/html/classvtkXMLWriter.html
So there's probably nothing we can do about that. Should we add a comment to the docs about that? In practice the precision of 11 shouldn't be a problem. There only might be some issues if somebody saves some simulation results to Ascii and then tries to start a second simulation from those results, applying too strict mesh searcher tolerances.